.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+python:
+  - "2.7"
+
+sudo: required
+
+services:
+  - docker
+
+before_install:
+  - sudo apt-get -y install curl python-software-properties software-properties-common python-pip python-dev build-essential git make
+  - sudo apt-get install -y libkrb5-dev
+  - sudo curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+  - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+  - sudo apt-get update
+  - sudo apt-get -y install docker-ce
```

Comment:
it is bunch workarounds what I've tried to make it work :-) after it will be working I'll remove all not necessary parts.